<launch>

	<!-- Arguments -->
	<param name="/use_sim_time" value="true"/>

	<!-- Launch ROAD LAYOUT ESTIMATION node -->
	<node
		pkg="road_layout_estimation" type="road_layout_estimation" name="road_layout_estimation"
		required="true"
		output="screen"
		args="stereo_odometer/odometry"
	/>
	
	</launch>
